app/talk/lib/paginator.cjsx

```diff
@@ -31,6 +31,9 @@ module?.exports = React.createClass
 
   contextTypes:
     geordi: React.PropTypes.object
+  
+  componentDidMount: ->
+    @logClick = @context.geordi?.makeHandler 'change-page'
 
   componentWillReceiveProps: (nextProps, nextContext)->
```

Comment:
I wonder if we should remove this now - not sure it actually achieves anything now that the `logClick` handler is defined in `componentDidMount` (which always happens earlier)